lib/shopify-cli/commands.rb

```diff
@@ -8,23 +8,23 @@ module Commands
     )
     @core_commands = []
 
-    def self.register(const, cmd, path = nil)
+    def self.register(const, cmd, path = nil, is_core = false)
       autoload(const, path) if path
       Registry.add(->() { const_get(const) }, cmd)
-      @core_commands.push(cmd)
+      @core_commands.push(cmd) if is_core
     end
 
     def self.core_command?(cmd)
       @core_commands.include?(cmd)
     end
 
-    register :Connect, 'connect', 'shopify-cli/commands/connect'
-    register :Create, 'create', 'shopify-cli/commands/create'
-    register :Help, 'help', 'shopify-cli/commands/help'
-    register :LoadDev, 'load-dev', 'shopify-cli/commands/load_dev'
-    register :LoadSystem, 'load-system', 'shopify-cli/commands/load_system'
-    register :Logout, 'logout', 'shopify-cli/commands/logout'
-    register :System, 'system', 'shopify-cli/commands/system'
-    register :Update, 'update', 'shopify-cli/commands/update'
+    register :Connect, 'connect', 'shopify-cli/commands/connect', true
```

Comment:
What do you think about using a keyword arguments to indicate what `true` means, e.g.

```ruby
register :Connect, 'connect', 'shopify-cli/commands/connect', core: true
```